src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,79 @@
+package homework_2.random_chars_table;
+
+import java.util.InputMismatchException;
+import java.util.Random;
+import java.util.Scanner;
+
+public class RandomCharsTable {
+
+    private static int length;
+    private static int width;
+
+    public static void run() {
+        System.out.println("Enter valid table length, width and strategy(even or odd) in format \"length width strategy\" ");
+        getParameters();
```

Comment:
Wrong naming

Reply:
You mean name for getParameters() method? Should I rename it to getInput() instead, is it ok?

Reply:
First of all, get methods usually return something. Second, the name of your method does not correlate with what it does. I would really split this method on logic blocks and call them in run() separately. These blocks could be like: getInput(); createTable(); printTable(); smth like this